apstra/two_stage_l3_clos_configlets.go

```diff
@@ -0,0 +1,144 @@
+package apstra
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	apiUrlBlueprintConfiglets       = apiUrlBlueprintById + apiUrlPathDelim + "configlets"
+	apiUrlBlueprintConfigletsPrefix = apiUrlBlueprintConfiglets + apiUrlPathDelim
+	apiUrlBlueprintConfigletsById   = apiUrlBlueprintConfigletsPrefix + "%s"
+)
+
+type TwoStageL3ClosConfiglet struct {
+	Configlet rawConfigletRequest `json:"configlet"`
+	Id        string              `json:"id"`
+	Condition string              `json:"condition"`
+	Label     string              `json:"label"`
+}
+
+func (o *TwoStageL3ClosClient) getAllConfiglets(ctx context.Context) ([]TwoStageL3ClosConfiglet, error) {
+	blueprintconfigleturl := fmt.Sprintf(apiUrlBlueprintConfiglets, o.blueprintId.String())
```

Comment:
variable isn't needed. Just put the `fmt.Sprintf()` in the only place it's used. Then we won't have to talk about capitalization ;)